### PR TITLE
docs(whitepaper): add use-cases of car-trade, electricity-trade, and supply-chain

### PR DESCRIPTION
**Pull request by takeutak**
Related with Issue #817

Signed-off-by: Takuma TAKEUCHI <takeuchi.takuma@fujitsu.com>

### CODE REVIEW

File: whitepaper/whitepaper.md [hunk 1]
```diff
@@ -371,14 +374,63 @@ The following table summarizes the use cases that will be explained in more deta
...
+| Success Criteria           | TBD |
+| Failure Criteria           | TBD |
+| Prerequisites              | TBD |
+| Comments                   |     |
+
+## 3.3 Supply chain
+
+| Use Case Attribute Name    | Use Case Attribute Value                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| Use Case Title             | Supply Chain |
```
Comment (petermetz) [inline whitepaper/whitepaper.md:415]: @takeutak Did you want me to fill this one out or are you still working on it?

**Review by petermetz — COMMENTED** [review]
@takeutak If you rebase to the latest main the tests should start passing on the CI

File: whitepaper/whitepaper.md (hunk 1 above)
Comment (takeutak) [reply whitepaper/whitepaper.md:415]: @petermetz I made a section template first In this PR. I think filling the contents of the table should be done by another PR.

Comment (petermetz) [reply whitepaper/whitepaper.md:415]: @takeutak Understood, thank you for clarifying!

**Review by petermetz — APPROVED** [review]

**Review by sfuji822 — APPROVED** [review]
LGTM

**Comment by petermetz** [comment]
@takeutak FYI: If you rebase to the latest main the CI should start passing after that.

**Pull request merged**